android-ia/device-androidia
# f2fs filesystem support enabled

f2fs filesystem support is enabled for using in adoptable storage
as per google AOSP CDD adoptable storage shall use both ext4 and f2fs. so enabling f2fs

Jira: None
Test: mount cmd
      1. create a storage volume with mkfs.f2fs utility
      2. mount -t f2fs <dev entry> <mount point>
      3. check mount cmd output and check for mount type as f2fs

Signed-off-by: shyjumon <shyjumon.n@intel.com>
